draft-tuexen-opsawg-pcapng.xml

```diff
@@ -2037,6 +2042,219 @@ Section Header
+        Combining pcap files with different location formats should result in the 
+        removal of all position information.
+        </t>
+
+        <t>
+        Velocity and Orientation are to be interpreted from the zero frame for either form.
+        For the global form, X corresponds to latitude, Y to longitude, and Z to altitude.
+        Yaw is about the X/Y or Lat/Lng plane with zero facing along Y or North, 
+        pitch is applied with this yaw offset, and roll is applied to the combination of both.
+        </t>
+
+        <t>
+        Error options provide an optional mechanism for attaching error information to measurements,
+        and are only valid where the corresponding measurement (location, velocity, or orientation) 
+        option is specified. Should an error option be present without the corresponding measurement, 
+        this option should be ignored.
+        </t>
+
+        <t>All the information fields are defined as options in order to support
+        systems that do not have a complete set of information. Therefore,
+        In addition to the options defined in <xref target="section_opt"/>,
+        the following options are valid within this block:</t>
+
+        <texttable anchor="options_cib" title="Capture Information Block Options">
+          <ttcol align="left">Name</ttcol>
+          <ttcol align="left">Code</ttcol>
+          <ttcol align="left">Length</ttcol>
+          <ttcol align="left">Multiple allowed?</ttcol>
+
+          <c>local_location</c>
+          <c>2</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>local_location_error</c>
+          <c>3</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>global_location</c>
+          <c>4</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>global_location_error</c>
+          <c>5</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>orientation</c>
+          <c>6</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>orientation_error</c>
+          <c>7</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>velocity</c>
+          <c>8</c>
+          <c>12</c>
+          <c>no</c>
+
+          <c>velocity_error</c>
+          <c>9</c>
+          <c>12</c>
+          <c>no</c>
+        </texttable>
+
+        <t>
+          <list hangIndent="8" style="hanging">
+
+            <t hangText="cib_local_location"><vspace blankLines="0"/>The 
+            cib_local_location option specifies the local location of the packet 
+            capture or interface in an abitrary plane; location is stored as three 
+            32-bit floats representing X, Y (in plane), and Z (altitude) offsets in
+            metres.</t>
+
+            <t>Example: '41 24 cc cd 41 73 33 33 3f cc cc cd' decodes to 
+            x: 10.3m, y: 15.2m, z: 1.6m</t>
+
+            <t hangText="cib_local_location_error"><vspace blankLines="0"/>The
```

Comment:
Perhaps it should indicate that having a cib_local_location_error option without a cib_local_location makes no sense, and that having a cib_local_location option without a cib_local_location_error option means that the error is unknown.

If so, the same should be done for the other options as well.

Reply:
I added that an overarching comment at line [2125](https://github.com/pcapng/pcapng/blob/5df4b9d9f6b4d7e902517a6e2c917479e8b02556/draft-tuexen-opsawg-pcapng.xml#L2125) rather than adding it per-option, though I can put it on each option if you would prefer?

Reply:
A global comment suffices.

What's the correct interpretation of a measurement without a corresponding error?  "It's unknown how accurate this measurement is"?

Reply:
I think the absence of error information only indicates that there is no error information.
As to whether it is unknown, unknowable, elided because the capturer doesn't care, or for any other reason, would be outside the scope of the capture file?